src/sass/_about.scss

```diff
@@ -0,0 +1,102 @@
+@import 'setup';
+@import 'variables';
+@import 'common';
+
+h2 {
+  margin: 0;
+  margin-block-end: 0;
+  margin-block-start: 0;
+}
+.about {
+  padding: 125px 0 39px 0;
+  background-color: getColor('background-light-pink');
+  &__article {
+    padding: 46px 30px 17px 30px;
+    font-family: getFont('font-main');
+    font-size: 12px;
+    font-weight: getFontWeight('font-medium');
+    line-height: 1.62;
+    color: getColor('dark-font');
+    max-width: 87%;
+  }
+  &__cow-image {
+    display: block;
+    margin: 0 auto;
+  }
+  &__milk-image {
+    background-image: url('../images/about/milk-bg-mobile.png');
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: bottom;
+  }
+  @media screen and (min-device-pixel-ratio: 2), (min-resolution: 192dpi), (min-resolution: 2dppx) {
+    &__milk-image {
+      background-image: url('../images/about/milk-bg-mobile@2x.png');
+      background-size: contain;
+      background-repeat: no-repeat;
+      background-position: bottom;
+    }
+  }
+}
+.title__label {
+  font-family: getFont('font-header');
+  font-size: 18px;
+  color: getColor('secondary-pink');
+  text-transform: uppercase;
+  align-self: center;
+  padding: 0 31px;
+  margin: 0 auto;
+}
+.section__title {
```

Comment:
W klasie section__title :  
  max-width: 65%;  zamienić na max-width: 200px ;
oraz dodać text-align: center ; 
Powinno to się wtedy nie rozjeżdżać i zostać ładnie na środku